src/Core/EspWindow.cc

```diff
@@ -56,6 +61,10 @@ namespace esp
     }
     glfwSetWindowUserPointer(m_window, &m_data);
 
+    // TODO: set flag for cursor disabled
+    // glfwSetInputMode(m_window, GLFW_CURSOR, GLFW_CURSOR_DISABLED);
```

Comment:
Maybe we could wrap this in an EspWindow method? Might be useful for the user

Reply:
Yeah, that's why I left a comment here